job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/VersionNumberContext.groovy

```diff
@@ -0,0 +1,92 @@
+package javaposse.jobdsl.dsl.helpers.wrapper
+
+import javaposse.jobdsl.dsl.Context
+
+class VersionNumberContext implements Context {
+
+    String format
+    String nameVariable
+    String prefixVariable
+    boolean skipFailedBuilds
+    boolean displayBuildName
+    String startDate
+    String buildsToday
+    String buildsThisWeek
+    String buildsThisMonth
+    String buildsThisYear
+    String buildsAllTime
+
+    /**
+     * @param format Version Number Format String
```

Comment:
Have a look at other contexts for the expected GroovyDoc. Use one sentence to describe the option and do not use the `@param` tags.